windows/CodePushUnknownException.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
```

Comment:
I don't think we need this exception type since we don't explicitly catch it anywhere, and we don't store any custom metadata along with the error message either.